### ISSUE TITLE
[SUGGESTION] Add some build challenges in the server for the pack

### ISSUE THREAD

**Issue by sshibacaretaker** [Gliese-832-c/ad-astra-per-nucleon#7]
###Build Challenges

I suggest making more challenges for the server. Kinda like the eVault where there is some build challenges. I propose to make some challenges on how to automate items that is hard to craft.

**Comment by Gliese-832-c** [comment]
I don't think that there are enough people playing AAPN for the challenges to make sense. How about you instead get a few hundred people to play the pack so we have at least a handful of people who would be willing to participate in the challenges? 😉

**Comment by sshibacaretaker** [comment]
_Sure thing_
